Makefile

```
@@ -57,6 +55,15 @@ clean-all: clean
 clean-all: ## Delete (almost) everything that can be reconstructed later
 	$(RM) -r *.egg-info/
 
+.PHONY: clean-build
+clean-build: ## Remove build artifacts
+	$(RM) -rf .eggs/
```

`$(RM)` already includes `-f` (but not `-r`).

```suggestion
	$(RM) -r .eggs/
```